<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Editor de imagens</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>17</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selecionar imagens</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>400</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selecione o filtro para remoção de ruído</string>
    </property>
   </widget>
   <widget class="QComboBox" name="equalizationBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>437</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Equalização de Histograma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equalização CLAHE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>437</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selecione a técnica para ajuste de contraste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>494</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aplicar configuração</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salvar Imagens</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>400</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Transformada de Wavelet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Filtro de Wiener</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Filtro de Mediana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Difusão Anisotrópica</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="imageList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>150</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>66</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="title">
     <string>Com ajuste de contraste</string>
    </property>
    <widget class="QLabel" name="imgEqualizada">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="text">
      <string>Equalizada</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>66</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="title">
     <string>Imagem original</string>
    </property>
    <widget class="QLabel" name="imgOriginal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Imagem Original</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>66</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="title">
     <string>Com remoção de ruído</string>
    </property>
    <widget class="QLabel" name="imgFiltrada">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="text">
      <string>Com tratamento</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
